nevergrad/optimization/oneshot.py

```diff
@@ -380,3 +386,49 @@ def __init__(
 AvgLHSSearch = SamplingSearch(sampler="LHS", recommendation_rule="average_of_best").set_name("AvgLHSSearch", register=True)
 AvgCauchyLHSSearch = SamplingSearch(sampler="LHS", cauchy=True, recommendation_rule="average_of_best").set_name(
     "AvgCauchyLHSSearch", register=True)
+
+# Recommendation rule = average of k best, with k = n / 2^d, i.e. average exponentially decreasing with the dimension.
+EAvgHaltonSearch = SamplingSearch(recommendation_rule="average_of_meunier_best").set_name("EAvgHaltonSearch", register=True)
```

Comment:
All those should now go to `experimentalvariants.py`

Reply:
Done.